packages/calcite-components/src/components/alert/alert.scss

```diff
@@ -68,13 +72,13 @@ $border-style: 1px solid var(--calcite-color-border-3);
   @apply flex flex-col items-center justify-center p-0;
   margin-block: auto;
   margin-inline-end: auto;
-  padding-inline-start: var(--calcite-alert-spacing-token-large);
+  padding-inline-start: var(--calcite-internal-alert-spacing-token-large);
```

Comment:
Why does this token exist? Usually component tokens don't have scale

Reply:
I'm not sure of why this token is there, I renamed it to internal because it looked like it's not something that will be set by users. Should I get rid of it and apply the spacing directly to the classes where this token is being used?

Reply:
It might help give visibility into what's happening.